pkgs/development/ocaml-modules/wtf8/default.nix

```
@@ -6,7 +6,7 @@ buildDunePackage rec {
 
   useDune2 = true;
```

```suggestion
  duneVersion = "2";
```
This can also likely be 3